.github/workflows/test_packages.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Test packages
+
+on:
+  workflow_dispatch:
+    inputs:
+      sharktank_package_version:
+        description: "Version of the sharktank package"
+        required: true
+      shortfin_package_version:
+        description: "Version of the shortfin package"
+        required: true
+
+jobs:
+  test_packages:
+    name: "Test packages"
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Setup python
+        uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.3
+        with:
+          python-version: 3.11
+      - name: Install Python packages
+        run: |
+          python -m pip install sharktank==${{ github.event.inputs.sharktank_package_version }} \
+            --no-deps -f https://github.com/nod-ai/SHARK-Platform/releases/expanded_assets/dev-wheels
+          python -m pip install shortfin==${{ github.event.inputs.shortfin_package_version }} \
+            --no-deps -f https://github.com/nod-ai/SHARK-Platform/releases/expanded_assets/dev-wheels
```

Comment:
Why `--no-deps`? Imports will likely fail if the packages don't also download their dependencies.

Reply:
If we want to test a thing like `from shartank import ops`, that's true. For what is tested right now I would like to avoid downloading torch and especially all the deps that get pulled in by torch.

Reply:
We need to be testing what users will do.

Reply:
I removed `--no-deps`.